src/Sarif.FunctionalTests/SpecExamples/RuleMetadata.sarif

```diff
@@ -0,0 +1,17 @@
+{
+  "version": "1.0.0",
+  "runs": [
+    {
+      "tool": {
+        "name": "CodeScanner",
+        "semanticVersion": "2.1.0"
+      },
+      "rules": {
+        "C2001": {
+          "id": "C2001",
+          "fullDescription": "A variable was used without being initialized. This can result in runtime errors such as null reference exceptions"
```

Comment:
Sounds good. I made a note in the Annex to expand the rules metadata example. I'll update it here once the Annex is accepted.